packit_service/worker/reporting.py

```diff
@@ -300,23 +325,46 @@ def set_status(
 class StatusReporterGithubChecks(StatusReporterGithubStatuses):
     project_with_commit: GithubProject
 
+    @staticmethod
+    def _create_table_content(
+        url: str, links_to_external_services: Optional[Dict[str, str]]
+    ) -> str:
+        table_content = []
+        if url:
+            table_content.append(f"| Dashboard | {url} |\n")
+        if links_to_external_services is not None:
+            table_content += [
+                f"| {name} | {link} |\n"
+                for name, link in links_to_external_services.items()
+            ]
+
+        return "".join(table_content)
+
     def set_status(
         self,
         state: BaseCommitStatus,
         description: str,
         check_name: str,
         url: str = "",
+        links_to_external_services: Optional[Dict[str, str]] = None,
     ):
         state_to_set = self.get_check_run(state)
         logger.debug(
             f"Setting Github status check '{state_to_set.name}' for check '{check_name}':"
             f" {description}"
         )
-        summary = (MSG_MORE_DETAILS.format(url=url) if url else "") + (
-            MSG_RERUN_NOT_SUPPORTED
-            if state_to_set == GithubCheckRunResult.failure
-            else ""
+
+        table_content = self._create_table_content(url, links_to_external_services)
+        summary = (
+            (MSG_TABLE_HEADER_WITH_DETAILS if table_content else "")
```

Comment:
wouldn't it be better to move this to `_create_table_content` (and rename the method) so that this method would manage the whole table creation?